### ISSUE TITLE
Not all dependencies are mentioned in the setup wiki

### ISSUE THREAD

**Issue by koalagang** [tam-carre/ncmpcpp-ueberzug#16]
The setup page mentions that ueberzug is needed but not other important packages. This was an issue for me because I had ueberzug, ffmpeg and ncmpcpp rightly installed but I was not aware that mpc was required (which I discovered upon inspecting the script).

I suggest listing the following dependencies on the setup page:
* ueberzug (as it already does, of course)
* ncmpcpp
* mpc
* ffmpeg

I'm aware that these may seem obvious but the point of a setup page is to make it crystal clear how to install a piece of software so I think it's worth mentioning them all.

**Comment by dextruct0r** [comment]
If you are using Void Linux there's +1 dependency needed to work. But it's needed the [script fix on the other post](https://github.com/tam-carre/ncmpcpp-ueberzug/issues/17#issuecomment-1229211325)

flac

**Comment by koalagang** [comment]
@dextruct0r I'm not using Void. I made this issue because I had to exam the script myself to discover all the dependencies. My intention with this issue was to address the fact that there is indeed a setup guide but I think, as with all software projects, it should be made as easy as possible to install. A user should not have to exam the codebase of a project to discover what they need to install to get it fully setup. If the [Setup](https://github.com/tam-carre/ncmpcpp-ueberzug/wiki/Setup) wiki page just had a list of the project's dependencies (i.e. the packages I listed in the opening comment) then the end-user would be able to install all those packages through their system package manager and then ncmpcpp-ueberzug would work perfectly well after following the other setup instructions.
